web/app/themes/justicejobs/search.php

```diff
@@ -686,7 +671,7 @@ class="search_contain__radio search_contain__radio--map"
                 </table>
             </div>
 
-            <div class="search_contain__map-wrap">
+            <div class="search_contain__map-wrap" >
```

Comment:
Maybe your IDE is set up incorrectly or this was a typo? I would suggest using an IDE that monitors and corrects HTML markup, to save you having to worry about added white space and formatting.

Reply:
This was probably me - I did a of adding and removing classes and ids in trialing something that would work, and didn't remove it. I'll look into setting a linter up in my IDE :)